### PR TITLE
[stable/grafana] Set resources for downloadDashboards initContainer

### PR DESCRIPTION
**Pull request by bclermont**
#### What this PR does / why we need it:

Add resource to an init container.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/grafana/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: grafana
-version: 5.0.6
+version: 5.0.7
```
Comment (zanhsieh) [inline stable/grafana/Chart.yaml]: Should be 5.0.9 for now.

**Review by zanhsieh — CHANGES_REQUESTED** [review]

**Comment by bclermont** [comment]
I rebased and bump version of chart to prevent conflict

**Comment by zanhsieh** [comment]
@bclermont 
Sorry need to bump again to latest.

**Comment by bclermont** [comment]
@zanhsieh done

**Pull request merged**